PowerPointLabs/Test/FunctionalTest/ShapesLabTest.cs

```diff
@@ -138,27 +135,25 @@ private void TestSavePlaceholderToShapesLabWithAddShapesButton(IShapesLabControl
             PpOperations.SelectSlide(shapesSlideNum);
             ExpectAddShapeButtonDisabled(shapesLab);
             PpOperations.SelectShapesByPrefix("Placeholder");
-            ExpectAddShapeButtonEnabled(shapesLab);
+            ExpectAddShapeButtonDisabled(shapesLab);
```

Comment:
is the function `ExpectAddShapeButtonEnabled(shapesLab)` unused in this script?

Reply:
line 171

Reply:
@leeyh20 I believe it is used. But selecting the (empty) placeholder does not seem to enable the button.